api/terraform/data_api/domain.tf

```diff
@@ -1,12 +1,17 @@
-data "aws_route53_zone" "dotorg" {
+/*data "aws_route53_zone" "dotorg" {
   provider = "aws.routemaster"
 
   name = "wellcomecollection.org."
+}*/
+
+locals {
+  route53_zone_id = "Z3THRVQ5VDYDMC"
 }
```

Comment:
This seems to have broken with the newer terraform version/provider:

> Error: Error finding Route 53 Hosted Zone: AccessDenied: User: arn:aws:sts::250790015188:assumed-role/wellcomecollection-assume_role_hosted_zone_update/1585053181121301000 is not authorized to perform: route53:ListTagsForResource on resource: arn:aws:route53:::hostedzone/Z3THRVQ5VDYDMC

😿

Reply:
It's not on the permissions for that role as of now - just looked.